src/summary/documentation/dimensions-card-row.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styles from "./dimensions-card-row.css";
+export default function DimensionsSectionCard(
+  props: DimensionsSectionCardProps
+) {
+  return (
+    <div className={styles.gridContainer}>
+      <div>{props.content.date}</div>
+      <div className={styles.text}>
+        {props.content.cm}
+        {props.index === 0 && <p>cm</p>}
+      </div>
+      <div className={styles.text}>
+        {props.content.kg}
+        {props.index === 0 && <p>kg</p>}
+      </div>
+      <div className={styles.text}>
+        {props.content.bmi}
+        {props.index === 0 && <p>m/m</p>}
```

Comment:
Should this be `kg/m`?? I'm looking at https://www.figma.com/file/YyVhWTNp89bNiOGadYrHBk/Patient-Chart---Master-(current-file)?node-id=1550%3A30065

```suggestion
        {props.index === 0 && <p>kg/m</p>}
```